### PR TITLE
Completed css-media-queries exercise. - Jeff Jin

### PR DESCRIPTION
**Pull request by jeffmjin**
https://user-images.githubusercontent.com/110066310/189153074-186c3447-0e37-4eae-8f12-6778d7f2c2aa.mov

https://user-images.githubusercontent.com/110066310/189153087-80b54032-ef30-4ec8-b91d-41a198e77c38.mov

https://user-images.githubusercontent.com/110066310/189153104-569cb2f8-d148-4668-8d26-aa51c3d6d402.mov

### CODE REVIEW

File: css-media-queries/meta-viewport.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <!-- your code here 👇 -->
+  <meta name="viewport" content="width=568">
```
Comment (BrettAlbright) [inline css-media-queries/meta-viewport.html]: make sure you are using the standard tag

File: css-media-queries/meta-viewport.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    .box {
+      width: 200px;
+      height: 200px;
+      background-color: blue;
+    }
+
+    @media only screen
+      and (orientation: landscape) {
```
Comment (BrettAlbright) [inline css-media-queries/meta-viewport.html]: this should be based on width of 576 px

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**